test/kmedoids.jl

```diff
@@ -39,6 +39,11 @@ R = kmedoids(costs, k)
 @test isapprox(sum(R.acosts), R.totalcost)
 @test R.converged
 
+# k=1 and k=n cases
+x = pairwise(SqEuclidean(), [[ 1 2 3]; [.1 .2 .3]; [4 5.6 7]], dims=2)
+@test nclusters(kmedoids(x, 1)) == 1
```

Comment:
You may need to tweak the coordinates that you use to generate `x` distance matrix, so that the choice of medoid point is more obvious, and that its index is different from 1 (the last test would need to be fixed then).
```suggestion
kmed1 = kmedoids(x, 1)
@test nclusters(kmed1) == 1
@test assignments(kmed1) == [1, 1, 1]
@test kmed1.medoids == [1]
```

Reply:
I hope that use of Set was an acceptable alternative :)